cfgov/v1/atomic_elements/atoms.py

```diff
@@ -26,6 +26,14 @@ class Meta:
 
 class Hyperlink(blocks.StructBlock):
     text = blocks.CharBlock(required=False)
+    aria_label = blocks.CharBlock(required=False,
+                                  help_text='Add an ARIA label if the link '
+                                            'text does not describe the '
+                                            'destination of the link '
+                                            '(e.g. has ambiguous text like '
+                                            '"Learn more" that is not '
+                                            'descriptive on its own).'
+    )
```

Comment:
Minor formatting suggestion:

```suggestion
    aria_label = blocks.CharBlock(
        required=False,
        help_text=(
            'Add an ARIA label if the link text does not describe the '
            'destination of the link (e.g. has ambiguous text like '
            '"Learn more" that is not descriptive on its own).'
        )
    )
```